doc/managing_wallet.md

```diff
@@ -0,0 +1,85 @@
+# Managing the Wallet
+
+## 1. Backing Up and Restoring The Wallet
+
+### 1.1 Creating the Wallet
+
+Since version 0.21, Bitcoin Core no longer has a default wallet.
+Wallets can be created with the RPC command `createwallet`. More information about this command can be found running `bitcoin-cli help createwallet`.
+
+The following command, for example, creates a descriptor wallet:
+
+`$ bitcoin-cli -named createwallet wallet_name="wallet-01" descriptors=true`
+
+The `descriptors` parameter can be omitted if the intention is to create a legacy wallet.
+
+For now, the default type is the legacy wallet, but that should change in the near future.
+
+Wallets are created in the `wallets` folder of the data directory. The default folder varies by operating systems, as shown below. The user can also change it using `-datadir` or `-walletdir` initialization parameters.
+
+| Operating System | Default wallet directory                                    |
+| -----------------|:------------------------------------------------------------|
+| Linux            | `home/<user>/.bitcoin/wallets`                              |
+| Windows          | `C:\Users\<user>\AppData\Roaming\Bitcoin\wallets`           |
+| macOS            | `/Users/<user>/Library/Application Support/Bitcoin/wallets` |
+
+### 1.2 Encrypting the Wallet
+
+The `wallet.dat` file is not encrypted by default and is, therefore, vulnerable if an attacker gains access to the device where the wallet or the backups are stored.
+
+The wallet must be encrypted with the following command:
```

Comment:
Re: encryption.

This changes past policy.  See @harding [here](https://github.com/bitcoin-core/gui/pull/96#issuecomment-705046732) "no released version of Bitcoin Core has ever created encrypted wallets by default...there's an open question between experts about whether or not the use of wallet encryption in typical user wallets saves more money than it loses" and @laanwj [here](https://github.com/bitcoin-core/gui/pull/96#issuecomment-705064355):  "I'm partial to not encrypting by default.  In any case the kind of scenario where encryption works is 'other people have physical access to my PC but won't use it to install a keylogger/backdoor'"

Different users will have different private key management and backup schemes, so I don't think it is appropriate to say that they must encrypt their keys.

Reply:
Is there a difference between encrypting **hot wallets** and **backups**?

Reply:
Maybe, but I don't believe so.  Again, it depends what the private key management policy is.  For multisig for example, I believe it is best to keep the backups unencrypted.  You get both security and redundancy with the m-of-n scheme.  With encryption, you run into issues of relying on memory or how to store the passphrases.

Reply:
Right. Better to make it clear that the decision is up to the user.